packages/fields/src/actions/duplicate-post.native.tsx

```diff
@@ -1,3 +0,0 @@
-const duplicatePost = undefined;
-
-export default duplicatePost;
```

Comment:
Why do we delete this file? Is it not used by mobile (I see it's named `*.native.*`)?